### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by FluffyDiscord** [sll552/DiscordBee#237]
Basically a title

**Comment by sll552** [comment]
I mean technically no as I have not planned on archiving this or not fixing stuff if something really (meaning it won't work anymore at all) breaks.

But I am of course aware that there has not been much activity and there are lingering issues especially with the image upload. This will must likely not change in the near future (not to be confused with ever again). 
For the reasons on why that's the case, I won't go much into detail but it's a mixture of:

* Personal stuff that I need to take care of. 
* Lack of motivation (I am not an as active user of Discord or MusicBee now, compared to when I started this project).
* Stress and other things that keep me from doing work/thought intensive things (including this) in my free time.

My hope was that maybe someone would be able to take over by making pull requests etc. but that never happened (which I am not mad about, but would have been a nice way to maybe get a second pair of eyes for the current issues)

**Comment by FluffyDiscord** [comment]
Have you swapped music players? If so, would you mind telling, or have you been converted to the streaming side :)

**Comment by sll552** [comment]
No, but I am not using it as much as a few years ago. This is the same for Discord.

It's still my music player of choice since winamp died 😉

**Comment by scapehips** [comment]
It still works fine tbh. If it ain't broke, don't fix it i guess.

**Comment by FluffyDiscord** [comment]
> It still works fine tbh. If it ain't broke, don't fix it i guess.

Unfortunately doesn't https://github.com/sll552/DiscordBee/issues/231

**Comment by sasspaper** [comment]
Take your time dude. Thanks for making this and sharing with us in the first place
